test/objects.mk

```diff
@@ -3,13 +3,22 @@
 #
 # Copyright (c) 2024 Ventana Micro Systems Inc.
 #
+#
+test-elfs-y += test_srv
```

Comment:
I still don't understand. Why separate server application ?

With this approach, each test scenario will require a separate test application.

Reply:
No, it doesnt, 

` process_ap_reqs()` inside `test_srv.c ` is common for all groups,
but server implementation definitely needs to be added just like we did in qemu for each service groups seperately as depicted below in  `add_*_group()` below:

```
 int main(int argc, char *argv[])
 {
         printf("Test server thread...\n");

         /*
          * along with contexts only BASE group is created, rest needs to be
          * added here later, after the context creation
          */
         create_contexts();
         
         add_hsm_group(&hsm);
         add_cppc_group(&cppc)
         
         process_ap_reqs();

         return 0;
 }

```

Reply:
This approach of adding a separate server for various scenarios is not going to scale. We have to re-design the common test infrastructure.